agentapi/agentapi.proto

```diff
@@ -10,7 +10,7 @@ service UI {
     rpc ApplyProToken (ProAttachInfo) returns (SubscriptionInfo) {}
     rpc ApplyLandscapeConfig(LandscapeConfig) returns (Empty) {}
```

Comment:
Fair point. I opted for returning the Landscape source. Not using that in the UI, for now at least, but we can think later of how to better use that return value and show some feedback in case the returned source is not `LandscapeSource.user`.